### PR TITLE
firebase-dsym-upload 1.0.0

### PR DESCRIPTION
**Pull request by andrewmarmion**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/firebase-dsym-upload/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- fdu_fabric_location: ./Pods/Fabric/upload-symbols
+  opts:
+    description: "This is the location of where Fabric is stored. \n"
+    is_expand: true
+    is_required: true
+    summary: Summary. No more than 2-3 sentences.
```
Comment (guness) [inline steps/firebase-dsym-upload/1.0.0/step.yml]:
```suggestion
```

File: steps/firebase-dsym-upload/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- fdu_fabric_location: ./Pods/Fabric/upload-symbols
+  opts:
+    description: "This is the location of where Fabric is stored. \n"
```
Comment (guness) [inline steps/firebase-dsym-upload/1.0.0/step.yml]:
```suggestion
    description: "This is the location of where Fabric is stored."
```

File: steps/firebase-dsym-upload/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  opts:
+    description: |
+      The locations is usually in the form ./YOUR-APP-NAME/GoogleService-Info.plist
+    is_expand: true
+    is_required: true
+    title: Location of your GoogleService-Info.plist
+- fdu_dsym_location: $BITRISE_DSYM_PATH
+  opts:
+    description: |
+      This is the locaiton of your dSYMs
```
Comment (guness) [inline steps/firebase-dsym-upload/1.0.0/step.yml]:
```suggestion
      This is the location of your dSYMs
```

File: steps/firebase-dsym-upload/1.0.0/step.yml (hunk 2 above)
Comment (andrewmarmion) [reply steps/firebase-dsym-upload/1.0.0/step.yml]: I have fixed this

File: steps/firebase-dsym-upload/1.0.0/step.yml (hunk 3 above)
Comment (andrewmarmion) [reply steps/firebase-dsym-upload/1.0.0/step.yml]: I have fixed this.

File: steps/firebase-dsym-upload/1.0.0/step.yml (hunk 1 above)
Comment (andrewmarmion) [reply steps/firebase-dsym-upload/1.0.0/step.yml]: I have fixed this.

File: steps/firebase-dsym-upload/1.0.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,55 @@
+title: Firebase-DSYM-Upload
```
[the remark this answers is not in the document]
Comment (andrewmarmion) [reply steps/firebase-dsym-upload/1.0.0/step.yml]: I have removed the `-` -s from the title as asked.

**Comment by richard-bogdan-bitrise** [comment]
Looks okay for me, @trapacska pls merge.

**Pull request merged**